src/topic/components/TrainersList.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
 import Text from '../../components/Text';
 import styled from 'styled-components';
-
+import PropTypes from 'prop-types';
+import {UserModel} from '../../shared/models/UserModel';
+import Trainer from '../../components/Trainer';
 
 const TrainersList = props => (
     <StyledTrainersList>
         <Text type="secondary" margin="0 0 0.5em 0">Trenerzy</Text>
-        <ul>
-            <li>
-                <Text>Piotr Zientara</Text>
-            </li>
-            <li>
-                <Text>Ula Kaźmierczyk</Text>
-            </li>
-            <li>
-                <Text>Kamil Gajowy</Text>
-            </li>
-        </ul>
```

Comment:
such a dream team of trainers removed in favour of John Does 😭

Reply:
Oh (John) Noes!